src/app/apptus.service.ts

```diff
@@ -0,0 +1,43 @@
+import {Injectable} from '@angular/core';
+import {HttpClient, HttpParams} from '@angular/common/http';
+import {Observable} from 'rxjs';
+
+const BASEURL = 'https://api.esales.apptus.com/clusters/w2AC5A25F/panels/';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApptusService {
+
+  constructor(private httpClient: HttpClient) {
+  }
+
+  public getCategoriesTree(): Observable<any> {
+    const url = 'navigation';
+    const params = new HttpParams()
+      .set('sessionKey', 'anonymous0')
+      .set('customerKey', 'anonymous0')
+      .set('market', 'UK');
+    const options = {params: params};
+    return this.httpClient.get(BASEURL + url, options);
+  }
```

Comment:
Can be simplified as follows:
```typescript
public getCategoriesTree(): Observable<any> {
    const params = { sessionKey: 'anonymous0', customerKey: 'anonymous0', market: 'UK' };
    return this.httpClient.get(`${ BASEURL }navigation`, { params });
  }
```

It is also recommended to use the data type instead of `any` because TS allows us to use types like in normal languages, so let's use this great opportunity :)